java/ql/test/kotlin/library-tests/classes/diags.ql

```diff
@@ -1,8 +0,0 @@
-import semmle.code.java.Diagnostics
```

Comment:
Is removing this test intended?

Reply:
Yes and no: It was actually removed in https://github.com/github/codeql/pull/13477, but this PR was accidentally on top of that branch, rather than on top of main. I'll rebase for clarity.